ftw/news/tests/test_news_listing_rss.py

```diff
@@ -84,3 +108,18 @@ def test_news_item_contains_pubdate(self, browser):
             news.news_date.strftime('%a, %-e %b %Y %H:%M:%S %z').strip(),
             browser.css('rdf item pubDate').first.text
         )
+
+        # Same test but with the view from "ftw.contentpage" for backward compatibility.
+        browser.login().visit(news_folder, view='news_rss_listing')
+
+        # Use HTML parser so that we have no XML namespaces.
+        browser.parse_as_html()
+
+        self.assertEqual(
+            # Difference between "%e" and "%-e":
+            # %e has a leading space on single numbers - that's why the tests
+            # were failing between the 1st and the 9th every month :-)
+            # %-e Removes the leading space - only works on unix machines.
```

Comment:
@mbaechtold FYI: http://stackoverflow.com/questions/28894172/why-does-d-or-e-remove-the-leading-space-or-zero 😄